### PR TITLE
add qhub-dask metapackage

### PR DESCRIPTION
**Pull request by prasunanand**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by costrouc** [comment]
I am okay with being a maintainer of this project

File: recipes/qhub-dask/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - dask ==2.14.0
+    - distributed ==2.14.0
+    - dask-gateway ==0.6.1
```
Comment (scopatz) [inline recipes/qhub-dask/meta.yaml:15]: Do these pins need to be so tight

Comment (prasunanand) [reply recipes/qhub-dask/meta.yaml:15]: Yeah.  We need a tight pinning for now.

**Comment by prasunanand** [comment]
ping @scopatz

**Comment by scopatz** [comment]
Thanks @prasunanand

**Pull request merged**